README.md

```diff
@@ -57,3 +57,26 @@ action "Create issue" {
   args = ".github/some-other-template.md"
 }
 ```
+
+#### Supported Custom Templates Structure:
+Template files should have the structure:
+```yaml
+---
+# Issue Parameters
+title: "A New Issue"
+labels:
+  - an-issue
+---
+# Issue Body
+Making a new issue can be automated!
+```
+
+#### Supported Issue Parameters (front matter):
+Field | Type | Description
+----- | ---- | -----------
+title | string | The title of the issue.
+labels | string[] | Labels to associate with this issue.
+milestone | integer | The `number` of the milestone to associate this issue with.
+assignees | string[] | Logins for Users to assign to this issue.
+name | string | Name of the template. _NOTE: Only used by the GitHub UI._
+about | string | Information describing this template. _NOTE: Only used by the GitHub UI._
```

Comment:
Maybe add a link to the official docs?

```suggestion
about | string | Information describing this template. _NOTE: Only used by the GitHub UI._

You can learn more about issue templates [from the GitHub docs](https://help.github.com/en/articles/manually-creating-a-single-issue-template-for-your-repository).
```

Reply:
I thought of that, however I didn't know how you felt about sending someone to a deprecated document.

Also with the yaml parameters and the actual create api parameters being out of sync, I didn't want to link a bunch and have the user be confused as to which they can and can't use.

Reply:
@JasonEtco what do you think?

Reply:
I don't love the idea of repeating documentation that is going to be "more right" somewhere else, especially if we're not linking to it. I'm not too concerned with the fact that that doc is deprecated - its only that way because it recommends doing it through the UI.